extensions/Cake.CachedNpm.yml

```diff
@@ -0,0 +1,16 @@
+Type: Addin
+Name: Cake.CachedNpm
+NuGet: Cake.CachedNpm
```

Comment:
This one looks like a candidate for the exclusion list. The repo doesn't seem to be anywhere on GitHub.

Reply:
Adding this addin to exclusion list: : https://github.com/cake-contrib/Cake.AddinDiscoverer/issues/194